src/Files.App/Views/Layouts/BaseLayoutPage.cs

```diff
@@ -15,6 +15,7 @@
 using System.IO;
 using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices.ComTypes;
+using Vanara.Extensions;
```

Comment:
What is this for?

Reply:
`NativeClipboard.CurrentDataObject.GetData` are in `Vanara.Extensions` now.

Reply:
Ah ok this is because of Vanara update, I'm fine with this change 👍